app/src/main/java/com/lastaoutdoor/lasta/ui/components/ShareSheets.kt

```diff
@@ -0,0 +1,23 @@
+package com.lastaoutdoor.lasta.ui.components
+
+import android.content.Context
+import android.content.Intent
+import com.lastaoutdoor.lasta.R
+import com.lastaoutdoor.lasta.models.activity.Activity
+
+// Opens the android share sheet, will contain a description and a link to the activity -> will be
+// used to share activities
+fun shareActivity(activity: Activity, context: Context) {
+
+  val sendIntent: Intent =
+      Intent().apply {
+        action = Intent.ACTION_SEND
+        putExtra(
+            Intent.EXTRA_TEXT,
+            context.getString(R.string.share_body) +
+                "\n${activity.name} \nhttps://lasta.jerem.ch/activities/${activity.activityId}")

```

Comment:
Really a nitpick but this url bit could also be in a string ressource in case we use it elsewhere

Reply:
Thank you for the remark, it is indeed a better choice and I will change this immediately